deploy/001-source-timelocks.ts

```diff
@@ -3,6 +3,7 @@ import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { SUPPORTED_NETWORKS } from "../helpers/deploy/constants";
+import { delayConfig } from "../helpers/deploy/deploymentConfig";
```

Comment:
The reasoning behind isolated configurations to deployments is similar to VIPs. The goal is to prevent leaking old values to new scripts.  One idea I've shared is to break deployment scripts up into 3 types. In this way if we need to run a setup script again we can easily see the original values and modify them without affecting other scripts.

1 - deployments
2 - configs with an id so they only run once ( and local configs)
3 - mocks

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/57/commits/efe36241ab5ad07fae501896b9139fd0aba265db